package.json

```diff
@@ -40,11 +40,11 @@
     "build:dev": "npm run remove-rcue; REACT_APP_RCUE=false npm run build:kiali",
     "build:kiali": "npm run lint:prod && npm run copy-fonts && npm run copy-img && npm run build-css && REACT_APP_VERSION=$npm_package_version REACT_APP_NAME=$npm_package_name REACT_APP_GIT_HASH=$(git rev-parse HEAD) react-scripts-ts build",
     "build:prod": "npm run restore-rcue; REACT_APP_RCUE=true npm run build:kiali",
-    "precommit": "(npm run prettier -- --list-different || (echo 'Above file(s) were modified by prettier, check them before commiting' && false) ) && npm run lint:prod",
+    "precommit": "yarn run pretty-quick -- --staged && npm run lint:prod",
     "copy-fonts": "cp -r node_modules/patternfly/dist/fonts src/",
     "copy-img": "cp -r node_modules/patternfly/dist/img src/",
     "lint": "tslint --project ./tsconfig.json",
-    "lint:prod": "tslint --project ./tsconfig.json --config tslint.prod.json",
+    "lint:prod": "if git diff --name-only HEAD | grep -E '\\.tsx?$'; then tslint --project ./tsconfig.json --config tslint.prod.json; else true; fi",
```

Comment:
I suppose this means that lint:prod will only run if there is any tsx?
Can we move it to a different entrypoint? `lint:prod-precommit` or put this if in the `precommit` ? 
I'm not sure if other parts of the pipeline use the lint:prod even if no file is modified.